docs/environment-variables.md

```diff
@@ -0,0 +1,33 @@
+# Environment variables
+
+In development, environment variables are resolved from the .env file
+In production you are expected to set environment variables during deployment (e. g. using Vault)
+
+## List of environment variables
+
+### app
+
+- `APP_NAME` - 
```

Comment:
I don't know what these four variables are for, any information on that would be appreciated